core/common/src/main/java/alluxio/security/authorization/AclEntry.java

```diff
@@ -110,6 +137,24 @@ public String toCliString() {
     return sb.toString();
   }
 
+  /**
+   * Convert a normal ACL to a string representing a default ACL.
+   * @param stringEntry normal ACL, if it is already default ACL, then return the default ACL
+   * @return a default ACL
+   */
+  public static String addDefault(String stringEntry) {
+    if (stringEntry == null) {
+      throw new IllegalArgumentException("Input acl string is null");
+    }
+    List<String> components = Arrays.stream(stringEntry.split(":")).map(String::trim).collect(
+        Collectors.toList());
+    if (components != null && components.size() > 0 && components.get(0).equals("default")) {
```

Comment:
Let's make `"default"` a constant, and use it everywhere in this file.

Reply:
done